apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/AllTypesTableFixture.cs

```diff
@@ -30,25 +30,53 @@ public AllTypesTableFixture() : base("TypesTable")
 
         protected override void CreateTable()
         {
-            ExecuteDdl($@"CREATE TABLE {TableName}(
-                            K                   STRING(MAX) NOT NULL,
-                            BoolValue           BOOL,
-                            Int64Value          INT64,
-                            Float64Value        FLOAT64,
-                            NumericValue        NUMERIC,
-                            StringValue         STRING(MAX),
-                            BytesValue          BYTES(MAX),
-                            TimestampValue      TIMESTAMP,
-                            DateValue           DATE,
-                            BoolArrayValue      ARRAY<BOOL>,
-                            Int64ArrayValue     ARRAY<INT64>,
-                            Float64ArrayValue   ARRAY<FLOAT64>,
-                            NumericArrayValue   ARRAY<NUMERIC>,
-                            StringArrayValue    ARRAY < STRING(MAX) >,
-                            BytesArrayValue     ARRAY < BYTES(MAX) >,
-                            TimestampArrayValue ARRAY<TIMESTAMP>,
-                            DateArrayValue      ARRAY<DATE>,
-                        ) PRIMARY KEY(K)");
+            // The emulator doesn't yet support the NUMERIC type.
+            if (RunningOnEmulator)
+            {
+                ExecuteDdl($@"CREATE TABLE {TableName}(
+                              K                   STRING(MAX) NOT NULL,
+                              BoolValue           BOOL,
+                              Int64Value          INT64,
+                              Float64Value        FLOAT64,
+                              StringValue         STRING(MAX),
+                              NumericValue        NUMERIC,
+                              BytesValue          BYTES(MAX),
+                              TimestampValue      TIMESTAMP,
+                              DateValue           DATE,
+                              BoolArrayValue      ARRAY<BOOL>,
+                              Int64ArrayValue     ARRAY<INT64>,
+                              Float64ArrayValue   ARRAY<FLOAT64>,
+                              NumericArrayValue   ARRAY<NUMERIC>,
+                              StringArrayValue    ARRAY < STRING(MAX) >,
```

Comment:
Are the spaces in these types necessary, or would `ARRAY<STRING(MAX)>` work?

Reply:
The spaces are not needed, so removed.